Version 4.6.1 (#69)
LycheeOrg/LycheeOrg.github.io: docs/releases.md

## Version 4

### v4.6.1

Released on Sep XX, 2022

#### IMPORTANT

This update will drop the API key in favor to Authorization token.

> As a result, once the migration is applied the API token will be revoked.

Reviewer: ```suggestion
As a result, once the migration is applied the old API token won't work anymore.
```

The token is not really revoked in a cryptographic or IT security sense.

## Version 4

### v4.6.1

Released on Sep XX, 2022

#### IMPORTANT

> - This update contains a Security Update which fix multiple XSS vulnerability and update the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).

Reviewer 2: ```suggestion
- This update contains a Security Update which fixes multiple XSS vulnerabilities and updates the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).
```

> - This update will drop the API key in favor to Authorization token.

Reviewer 2: ```suggestion
- This update will drop the API key in favor of Authorization token.
```

As a result, once the migration is applied the old API token won't work anymore.

#### Changes

* `new` #1489 : Drop page support
  > This functionality was not known by 99.9% of the users of Lychee.
  > We decided to drop its support in favor of more targetted development in the core.
* `new` #1443 : Add user and better structure to session json
* `new` #1153 : Support for multi path on server import
* `fixes` #1508 : Ensure that Admin rights are overloading others
* `fixes` #1366 : Improve config options
* `fixes` #1514 : 32 bits system improved support
  > For 32bit systems and time-based, legacy IDs we must catch failing inserts due to duplicate legacy ID

>   > when insertions is running to fast. In the path it was sufficient to check for MySQL error codes 23000

Reviewer 2: ```suggestion
  > when insertions is running too fast. In the past it was sufficient to check for MySQL error codes 23000
```
